building/building-archlinux-template.md

```diff
@@ -283,26 +283,26 @@ redirect_from:
 
             *   These are the indivual component 'make' commands:
 
-                *   **$ make vmm-xen-vm**
+                $ make vmm-xen-vm
 
-                *   **$ make core-vchan-xen-vm**
+                $ make core-vchan-xen-vm
 
-                *   **$ make core-qubesdb-vm**
+                $ make core-qubesdb-vm
 
-                *   **$ make linux-utils-vm**
+                $ make linux-utils-vm
 
-                *   **$ make core-agent-linux-vm**
+                $ make core-agent-linux-vm
 
-                *   **$ make gui-common-vm**
+                $ make gui-common-vm
 
-                *   **$ make gui-agent-linux-vm**
+                $ make gui-agent-linux-vm
```

Comment:
This should probably be a bulleted list of snippets.

Reply:
I was just suggesting that you turn it into a bulleted list of snippens, since I think doing so would be an improvement over the current state of affairs. Yes, it would be fine to merge anyway as long as the syntax is valid.

Reply:
**Sorry, let me clarify:**
I believe I have done that in one of the commits below.  But it didn't show up how I expected on the github preview.

Reply:
I see. The GitHub preview might not be accurate. See below for a link to instructions for rendering the site locally for a more accurate preview of what your changes will look like.